src/views/misc/AskUsView.vue

```diff
@@ -1,3 +1,246 @@
 <template>
-  <h1>Ask Us</h1>
+  <div class="ask-us">
+      <h1>Ask Us</h1>
+      <p>Have a question about the site? Don't know where to start? You can search topics in our 
+        <a href="https://conf.phoenixbioinformatics.org/display/MD/MorphoBank+Documentation" target="_blank">manual</a> 
+        by typing key words into your browser's search. Or contact us - we love to hear from your users.</p>
+      
+      <form @submit.prevent="submitForm">
+          <div class="form-group">
+              <label for="name">Name</label>
+              <input id="name" v-model="form.name" type="text">
+          </div>
+          <div class="form-group">
+              <label for="email">E-mail Address</label>
+              <input id="email" v-model="form.email" type="email">
+          </div>
+          <div class="form-group">
+              <label for="question">Tell us your question, issue or idea</label>
+              <textarea id="question" v-model="form.question"></textarea>
+          </div>
+          <div class="form-group">
+              <label for="media">Media or Matrix numbers affected (if applicable)</label>
+              <input id="media" v-model="form.media" type="text">
+          </div>
+          <div class="form-group">
+              <label for="projectNumber">Project Number (if applicable)</label>
+              <input id="projectNumber" v-model="form.projectNumber" type="text">
+          </div>
+          <div class="form-group checkbox">
+              <input id="published" v-model="form.published" type="checkbox">
+              <label for="published">Check here if this project is currently "published" on MorphoBank (visible to the public)</label>
+          </div>
+          <div class="form-group">
+              <label for="security">Security Question (to prevent SPAMbots): {{ firstNumber }} + {{ secondNumber }} = ?</label>
+              <input id="security" v-model="form.security" type="text">
+          </div>
+          <div class="form-group">
+              <label for="attachment">Related Attachments</label>
+              <input id="attachment" type="file" multiple @change="onFileChange">
+              <p class="warning-message">Attachments must not exceed a total size of 9MB.</p> <!-- Static warning message -->
+          </div>
+          <div class="form-group action-buttons">
+              <button type="submit">Submit</button>
+              <button type="reset">Reset</button>
+          </div>
+          <div v-if="errorMessage" class="alert alert-danger" role="alert">
```

Comment:
Please use the Alert component we created under the main/component repo to have uniformed Alert behavior.